flink-streaming-java/src/main/java/org/apache/flink/streaming/api/windowing/triggers/ContinuousProcessingTimeTrigger.java

```diff
@@ -80,6 +80,9 @@ public TriggerResult onProcessingTime(long time, W window, TriggerContext ctx) t
 			ctx.registerProcessingTimeTimer(time + interval);
 			return TriggerResult.FIRE;
 		}
+		if (window.maxTimestamp() == time) {
+            return TriggerResult.FIRE_AND_PURGE;
```

Comment:
To bring this in line with `ContinuousEventTimeTrigger` it should only be `FIRE`.